destinationCreation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8"/>
+      <title>Destination Creation</title>
+      <link rel="stylesheet" href="destinationCreation.css"/>
+      <script src="destinationCreation.js"></script>
+    </head>
+    <header>
```

Comment:
I think this should be removed since it's empty. Also I'm not sure if it's permitted outside of the `<body>`. But if some automated tool told you to put it in, that's fine.